Source/WebCore/Modules/mediastream/gstreamer/GStreamerRtpTransceiverBackend.cpp

```diff
@@ -109,7 +109,7 @@ bool GStreamerRtpTransceiverBackend::stopped() const
     return m_isStopped;
 }
 
-static inline WARN_UNUSED_RETURN ExceptionOr<GstCaps*> toRtpCodecCapability(const RTCRtpCodecCapability& codec, int payloadType)
+static inline WARN_UNUSED_RETURN ExceptionOr<GstCaps*> toRtpCodecCapability(const RTCRtpCodecCapability& codec, int* dynamicPayloadType)
```

Comment:
I think it makes more sense to use a `int& dynamicPayloadType`. That way you don't need the `*`  to access and even to modify the variable.

Reply:
Indeed, great idea!